### DISCUSSION TITLE
small visual bug

### DISCUSSION THREAD

**Discussion by danix1234** [home-sweet-gnome/dash-to-panel#1841] [General]
running indicators style "dots", "square" and "dashes" don't show when opening favorites apps
![Screenshot from 2023-03-05 17-43-08](https://user-images.githubusercontent.com/116650308/222973844-9e7d9e5f-3f4d-4cd8-abac-10aa766d4421.png)
as you can see in the image, gnome-terminal has no circle to indicate a window is open.

**Comment by charlesg99** [comment]
Thanks for reporting this! I'll wait till tomorrow to see if there are other issues from the latest release and make a new one with the fix.

**Reply by danix1234** [reply]
thanks for fixing it!

**Comment by fedkad** [comment]
Same problem here, _after update to version 53_. Sometimes they show, sometimes they don't. Opening the **Dash to Panel Settings** and switching the **Running indicator position** will refresh and show all missing running indicators. But, then again, after you close and open the application, its indicator is missing.
